java/core/generate-test-sources-build.xml

```diff
@@ -6,6 +6,7 @@
         <arg value="--proto_path=${test.proto.dir}"/>
         <arg value="${protobuf.source.dir}/google/protobuf/map_lite_unittest.proto"/>
         <arg value="${protobuf.source.dir}/google/protobuf/unittest.proto"/>
+        <arg value="${protobuf.source.dir}/google/protobuf/unittest_features.proto"/>
```

Comment:
Addresses the following build failure:

```
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /.../protobuf/java/core/src/test/java/com/google/protobuf/DescriptorsTest.java:[40,10] package pb does not exist
[ERROR] /.../protobuf/java/core/src/test/java/com/google/protobuf/DescriptorsTest.java:[1316,11] cannot find symbol
  symbol:   variable UnittestFeatures
```

<details>
<summary>Full error snippet</summary>
<pre>
[INFO] --- compiler:3.6.1:testCompile (default-testCompile) @ protobuf-java ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 180 source files to /.../protobuf/java/core/target/test-classes
[INFO] /.../protobuf/java/core/target/generated-test-sources/com/google/protobuf/UnittestLite.java: Some input files use or override a deprecated API.
[INFO] /.../protobuf/java/core/target/generated-test-sources/com/google/protobuf/UnittestLite.java: Recompile with -Xlint:deprecation for details.
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /.../protobuf/java/core/src/test/java/com/google/protobuf/DescriptorsTest.java:[40,10] package pb does not exist
[ERROR] /.../protobuf/java/core/src/test/java/com/google/protobuf/DescriptorsTest.java:[1316,11] cannot find symbol
  symbol:   variable UnittestFeatures
  location: class com.google.protobuf.DescriptorsTest.FeatureInheritanceTest
[ERROR] /.../protobuf/java/core/src/test/java/com/google/protobuf/DescriptorsTest.java:[1317,33] cannot find symbol
  symbol:   variable UnittestFeatures
  location: class com.google.protobuf.DescriptorsTest.FeatureInheritanceTest
[ERROR] /.../protobuf/java/core/src/test/java/com/google/protobuf/DescriptorsTest.java:[1323,36] cannot find symbol
  symbol:   variable UnittestFeatures
  location: class com.google.protobuf.DescriptorsTest.FeatureInheritanceTest
[INFO] 4 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Protocol Buffers [Parent] 4.27.0:
[INFO] 
[INFO] Protocol Buffers [BOM] ............................. SUCCESS [  0.001 s]
[INFO] Protocol Buffers [Parent] .......................... SUCCESS [  0.224 s]
[INFO] Protocol Buffers [Lite] ............................ SUCCESS [ 11.285 s]
[INFO] Protocol Buffers [Core] ............................ FAILURE [ 11.009 s]
[INFO] Protocol Buffers [Util] ............................ SKIPPED
[INFO] Protocol Buffers [Kotlin-Core] ..................... SKIPPED
[INFO] Protocol Buffers [Kotlin-Lite] ..................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  23.014 s
[INFO] Finished at: 2024-03-14T14:14:33-07:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.6.1:testCompile (default-testCompile) on project protobuf-java: Compilation failure: Compilation failure: 
[ERROR] /.../protobuf/java/core/src/test/java/com/google/protobuf/DescriptorsTest.java:[40,10] package pb does not exist
[ERROR] /.../protobuf/java/core/src/test/java/com/google/protobuf/DescriptorsTest.java:[1316,11] cannot find symbol
[ERROR]   symbol:   variable UnittestFeatures
[ERROR]   location: class com.google.protobuf.DescriptorsTest.FeatureInheritanceTest
[ERROR] /.../protobuf/java/core/src/test/java/com/google/protobuf/DescriptorsTest.java:[1317,33] cannot find symbol
[ERROR]   symbol:   variable UnittestFeatures
[ERROR]   location: class com.google.protobuf.DescriptorsTest.FeatureInheritanceTest
[ERROR] /.../protobuf/java/core/src/test/java/com/google/protobuf/DescriptorsTest.java:[1323,36] cannot find symbol
[ERROR]   symbol:   variable UnittestFeatures
[ERROR]   location: class com.google.protobuf.DescriptorsTest.FeatureInheritanceTest
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :protobuf-java
</pre>
</details>